lib/index.js

```diff
@@ -54,3 +56,8 @@ bedrock.events.on('bedrock.init', () => {
 function _invalidStateError() {
   return new BedrockError('The agent is not ready.', 'InvalidStateError');
 }
+
+// For testing purposes only
+function _resetHttpsAgent() {
```

Comment:
This doesn't reset the https agent, it clears it, setting it to `null`. Perhaps this should take a param that can be set to `null` and renamed to `setHttpsAgent` instead.

Reply:
Updated 29b8f78